android/quest/src/main/resources/cql/pq-medication/cql.txt

```diff
@@ -0,0 +1,135 @@
+//Declare name and version of lib
```

Comment:
is there an issue to move this be loaded instead of hard-coded?

Reply:
same w/android/quest/src/main/resources/cql/pq-medication/data.json

Reply:
and I think everything in this folder

Reply:
resources existing into main was confusing hence I moved all resources to test folder. These resources are not hardcoded rather used by unit tests to validate if cql is generating the right output entities with given data.. This all is just for unit tests and none is used in actual code. This helps ensure that if we update CQL it produces correct results. We had a discussion to add such on server side so that it is ensured that any updates to libraries, structure map, questionnaires are tested and validated

Reply:
cool thanks, yes putting these in the test folder makes this clear